tests/_utils/view.js

```diff
@@ -0,0 +1,836 @@
+/**
+ * @license Copyright (c) 2003-2016, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+/* jshint latedef:false */
```

Comment:
Yes - this is used to suppress the late class definition JSHint warning. Class is used in code that is located above definition of the class but instance is not created until execution of `parse` / `stringify` methods. I wanted to have my helper/private classes to be located later in the file. What is better then to avoid JSHint errors: to move classes to the beginning of the file or put them in separate files?